me.broken_by.PdfMetadataEditor.yml

```diff
@@ -0,0 +1,38 @@
+app-id: me.broken_by.PdfMetadataEditor
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+commands:
+  - "/app/bin/pmedit-cli"
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --filesystem=home
+  - --persist=.java
+modules:
+  - name: pdf-metadata-editor-app
+    buildsystem: simple
+    build-commands:
+      - mv -v * ${FLATPAK_DEST}
+    sources:
+      - type: archive
+        url: https://github.com/zaro/pdf-metadata-editor/releases/download/v3.0.0/pdf-metadata-editor-3.0.0-linux-portable.tar.gz
+        sha256: 80303e459c1c3b6ecf13923545ca349a2c3e1ff2a3ede08644adba98b2a9de2d
+  - name: pdf-metadata-editor-desktop
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps
+      - mkdir -p ${FLATPAK_DEST}/share/mime/packages/
+      - mkdir -p ${FLATPAK_DEST}/share/applications/
+      - mkdir -p ${FLATPAK_DEST}/share/metainfo/
+      - cp -v flatpak/pdf-metadata-edit.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/me.broken_by.PdfMetadataEditor.png
+      - cp -v flatpak/me.broken_by.PdfMetadataEditor-mime.xml ${FLATPAK_DEST}/share/mime/packages/me.broken_by.PdfMetadataEditor.xml
+      - cp -v flatpak/me.broken_by.PdfMetadataEditor.desktop ${FLATPAK_DEST}/share/applications/
+      - cp -v me.broken_by.PdfMetadataEditor.metainfo.xml ${FLATPAK_DEST}/share/metainfo/
```

Comment:
```suggestion
      - install -Dm644 flatpak/pdf-metadata-edit.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/me.broken_by.PdfMetadataEditor.png
      - install -Dm644 flatpak/me.broken_by.PdfMetadataEditor-mime.xml ${FLATPAK_DEST}/share/mime/packages/me.broken_by.PdfMetadataEditor.xml
      - install -Dm644 flatpak/me.broken_by.PdfMetadataEditor.desktop -t ${FLATPAK_DEST}/share/applications/
      - install -Dm644 me.broken_by.PdfMetadataEditor.metainfo.xml -t ${FLATPAK_DEST}/share/metainfo/
```

Reply:
also I shall add that none of these files are flatpak specific so wherever you have put them in the repoistory is confusing. Not that it matters here, but I have to debunk that myth.

Reply:
also the .desktop file, you didn't fix the `Exec` line. (you ignored the review comment)

Reply:
I for sure haven't heard about that myth , so no chance of it sticking on me now :)

Regarding the .desktop file , what do you mean ?
I did change it and removed the absolute path. The other comment was about the spaces in the binary name, but as I tried to explain in the other comment this is a limitation of the `jpackage` tool I am using to build the software.  I don't like it at all that there are spaces, but I am not sure what to do about it, because the tool doesn't have separate options for Application name and Binary name, you have to pick the same for both.

Is this a huge problem? it seems to work just fine.